tests/unit/test_isort.py

```diff
@@ -1629,13 +1629,13 @@ def test_multiline_import() -> None:
     assert isort.code(test_input) == ("from pkg import more_stuff, other_suff, stuff\n")
 
     # test again with a custom configuration
-    custom_configuration = {
+    custom_configuration: Dict[str, Any] = {
```

Comment:
Can you use `dict[str, Any]`?

Reply:
> Can you use `dict[str, Any]`?

yep, now that we don't support Python 3.8 that will work

Reply:
done with [cbd07e0](https://github.com/PyCQA/isort/pull/2329/commits/cbd07e010ebb94b37306e26d7d21cf85c2c05ae8)